tests/test_deps.py

```diff
@@ -30,13 +32,16 @@ def test_requirements_txt():
 # because they require the assets to be installed. In the future, it would make sense to
 # run this test when we're on an rc branch.
 # ======================================================================================
-if os.environ.get("CI") == "true" and os.environ.get("GITHUB_EVENT_NAME") != "release":
+if os.environ.get("CI") == "true":
```

Comment:
Temporarily deleting this line so we can actually test this without a release.

```suggestion
if os.environ.get("CI") == "true"  and os.environ.get("GITHUB_EVENT_NAME") != "release":
```